float64 tibia_LF
float64 femur_LF
float64 coxa_LF

float64 tibia_RF
float64 femur_RF
float64 coxa_RF

float64 tibia_LM
float64 femur_LM
float64 coxa_LM

float64 tibia_RM
float64 femur_RM
float64 coxa_RM

float64 tibia_LB
float64 femur_LB
float64 coxa_LB

float64 tibia_RB
float64 femur_RB
float64 coxa_RB